package.json

```diff
@@ -11,12 +11,14 @@
     "ansi_up": "^1.3.0",
     "bootstrap": "^3.3.7",
     "bootstrap-confirmation2": "^2.4.0",
-    "eonasdan-bootstrap-datetimepicker": "^4.17.43",
-    "jquery": "^2.2.4",
+    "bootstrap-datetime-picker": "^2.4.3",
+    "jasny-bootstrap": "^3.1.3",
```

Comment:
That provides the improved file input: http://www.jasny.net/bootstrap/javascript/#fileinput